synced_versions_formulae.json

```diff
@@ -32,7 +32,7 @@
   ["easyrpg-player", "liblcf"],
   ["etl", "synfig"],
   ["file-formula", "libmagic"],
-  ["edencommon", "fb303", "fbthrift", "fizz", "folly", "mvfst", "proxygen", "wangle", "watchman"],
+  ["edencommon", "fb303", "fatal", "fbthrift", "fizz", "folly", "mvfst", "proxygen", "wangle", "watchman"],
```

Comment:
Probably doesn't need to be added here -- they don't need to be updated together, and it just makes CI take longer when updating `edencommon`, `fb303`, etc.

Reply:
yeah, works for me.